### ISSUE TITLE
[$125] Keyboard Navigation: Many Pages: Several elements have no keyboard focus indicator

### ISSUE THREAD

**Issue by nbhardwaj-del** [Expensify/App#74763]
## Action Performed
1. Using any desktop browser, open the site https://www.expensify.com/terms
2. Press Tab key to navigate through the interactive elements in the page.
3. Note if the focused elements have a visible focus indicator.

Other Occurrences:
1. On Workspace - Reports: all buttons on the left side navigation menu

## Expected Result
Each interactive element should have a visible focus outline, for example, a dotted outline offset from the control, or a highly visible custom indicator so that the user can see what is focused.

## Actual Result
Multiple interactive elements do not have a visible focus indicator.

HTML:
<a href=""/karmaprogramterms"">Karma Program Terms</a>

CSS:
:focus { outline: 0; outline-offset: 0;}"

## Area issue was found
Terms of Service

## Failed Checkpoint
WCAG 2.4.7 Focus Visible
https://www.w3.org/WAI/WCAG22/Understanding/focus-visible

## User Impact
Users who rely on the keyboard to operate the page can visually determine the component on which keyboard operations will interact at any point in time. People with attention limitations, short term memory limitations, or limitations in executive processes benefit by being able to discover where the focus is located.

## Suggested Resolution
Allow the default keyboard focus outline to remain visible unless an alternative highly visible focus outline is provided. Any instances of outline:0 or outline:none applied should be removed. For custom controls provide a custom outline on the visible element if required.
https://www.w3.org/WAI/WCAG21/Techniques/general/G195

## Environment(s)
Windows Windows 10 - Chrome

<img width="1914" height="1004" alt="Image" src="https://github.com/user-attachments/assets/aa45ff80-f5b9-4e91-a33d-d908edeba90a" />

https://github.com/user-attachments/assets/e64a3bea-ac35-4d2f-8c51-84a6e4b60e10

**Version Number:** NA
**Reproducible in staging?:** NA
**Reproducible in production?:** Yes
**Email or phone of affected tester (no customers):** NA
**Logs:** https://stackoverflow.com/c/expensify/questions/4856 NA
**Notes/Photos/Videos:** Any additional supporting documentation - Yes
**Expensify/Expensify Issue URL:** https://platform.applause.com/company/12102/products/36605/community-issues/6985857
**Issue reported by:** NA
**Slack conversation:** NA

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~022009685681466575086</li>
        <li>Upwork Job ID: 2009685681466575086</li>
        <li>Last Price Increase: 2026-01-28</li>
    </ul>
</details>

<details><summary>Issue Owner</summary>Current Issue Owner: @mkhutornyi</details>

**Comment by rushatgabhane** [comment]
the correct amount for this issue is $125

**Comment by mkhutornyi** [comment]
@rushatgabhane this seems like OD issue

**Comment by rushatgabhane** [comment]
This issue was retested in **BFV cycle 544083 (Web)** on 2026-04-28 and is **Not Fixed**.

### Screenshots / Videos
- https://platform.applause.com/services/links/v1/external/81efb8f7c299447b6185defc005a25e3ce640c6d1dd288f1301938f678a5c141

### Retest notes
The issue is not fixed

Multiple interactive elements do not have a visible focus indicator.

Issue verified on: Windows 11
Browser: Chrome browser
Web URL/App Version if native app: https://www.expensify.com/terms
Please see the attached video.

**Comment by mkhutornyi** [comment]
@rushatgabhane this is OD issue so it should be internal
